build.gradle

```diff
@@ -56,6 +57,16 @@ task clean(type: Delete) {
 configurations {
     ktlint
 }
+// To ensure unit tests and instrumentation tests remain consistent after the introduction of leakcanary
```

Comment:
nit: I agree with the sentiment – let's not change the tests just yet. Can you:
- file an issue (you don't have to work on it) to investigate whether we want to add this to unit or UI tests
- Mention "See #xxx" in this comment where xxx is the issue I mention above.

Reply:
See #2041